src/ToolBox/superpmi/superpmi/commandline.cpp

```diff
@@ -113,6 +114,12 @@ void CommandLine::DumpHelp(const char* program)
     printf("     Use disassembly tools from the CoreDisTools library\n");
     printf("\n");
 #endif // USE_COREDISTOOLS
+    printf(" -jitoption key=value\n");
+    printf("     Set the JIT option named \"key\" to \"value\" for JIT 1. NOTE: do not use a \"COMPlus_\" prefix!\n");
+    printf("\n");
+    printf(" -jit2option key=value\n");
+    printf("     Set the JIT option named \"key\" to \"value\" for JIT 2. NOTE: do not use a \"COMPlus_\" prefix!\n");
+    printf("\n");
```

Comment:
The ability to specify different options to each JIT is the crux of this change: it allows a user to use the same JIT with different options for the base and the diff. This allows us to more easily experiment with JIT behavior that is governable by switches.